### DISCUSSION TITLE
通过URL导入配置文件时失败，显示EOF

### DISCUSSION THREAD

**Discussion by damaoooo** [MetaCubeX/ClashMetaForAndroid#285] [Q&A]
如题，在wIndows的Clash Meta Rev，NekoBoxForAndroid 可以正常导入，但是ClashMetaForAndroid导入失败，显示Get "<subscription link>" ，EOF

![QQ20240802-194616](https://github.com/user-attachments/assets/4a62edf3-1c3f-4798-9dbd-e98666487940)

如果有开发者可以告知相关处理的逻辑在何处就好了，我尝试自己编译并进行debug，发现找不到相关逻辑
service/src/main/java/com/github/kr328/clash/service/ProfileProcessor.kt:81 处有相关逻辑，但是我下断点后程序不暂停。

**Comment by felixreed** [comment]
CMFA的地址订阅似乎不会过系统代理，直连的，当订阅链接需要代理的时候，哪怕系统代理科学了，依然不能下载订阅

**Comment by saltFishCode** [comment]
大概率是该订阅的服务器被墙了，
我使用的clash meta for Android也出现部分订阅eof，
电脑的clash倒是没问题，
想办法让自己的手机开启代理（搞一些免费的垃圾vpn）然后再去导入订阅，值得一提的是我这边的规则模式还是无法有效，开全局才可以
